### DISCUSSION TITLE
Vercel Takeover no longer possible!!

### DISCUSSION THREAD

**Discussion by zangcc** [projectdiscovery/nuclei#5271] [Q&A]
Is this vulnerability no longer exploitable? Why does nuclei still display this script?
<img width="944" alt="image" src="https://github.com/EdOverflow/can-i-take-over-xyz/assets/64825932/54beb2bf-1769-4ef1-8073-ba8f64e741ca">

![image](https://github.com/projectdiscovery/nuclei-templates/assets/64825932/786271b3-f93c-423c-8312-755894a59605)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
